src/lua/BabblerBomb/BabblerBomb.lua

```diff
@@ -0,0 +1,103 @@
+Script.Load("lua/Weapons/Alien/Bomb.lua")
+Script.Load("lua/BabblerBomb/Bombler.lua")
+
+class 'BabblerBomb' (Bomb)
+
+BabblerBomb.kMapName            = "babbler_bomb"
+BabblerBomb.kModelName          = PrecacheAsset("models/alien/babbler/babbler_egg.model")
+kBomblerMaxNumber = 6
+BabblerBomb.kRadius             = 0.5
+
+local networkVars = { }
+
+function BabblerBomb:GetProjectileModel()
+    return BabblerBomb.kModelName
+end 
+
+if Server then
+
+   -- Babblers are 0.25 wide. Give ourselves some wiggle room
+    local kCheckDist = Babbler.kRadius + 0.25
+    local kVerticalOffset = kCheckDist
+    local kBabblerSpawnPoints =
+    {
+        Vector(kCheckDist, kVerticalOffset, kCheckDist),
+        Vector(-kCheckDist, kVerticalOffset, -kCheckDist),
+        Vector(0, kVerticalOffset, kCheckDist),
+        Vector(0, kVerticalOffset, -kCheckDist),
+        Vector(kCheckDist, kVerticalOffset, 0),
+        Vector(-kCheckDist, kVerticalOffset, 0),    
+    }
+    local babblerExtents = Vector(kCheckDist, kCheckDist, kCheckDist)
+    
+    function BabblerBomb:OnInitialized()
+        self.timeCreated = Shared.GetTime()
+    end
+
+    function BabblerBomb:ProcessHit(targetHit, surface, normal)
+
+        local ownerOk = false
+        if (Shared.GetTime() - self.timeCreated) > 2 then
+            ownerOk = true
+        end

```

Comment:
What is the purpose of this check? Also, would be more readable as a guard clause `if true then return end`